src/components/room/RoomCard.vue

```
@@ -0,0 +1,143 @@
+<script setup lang="ts">
+import type { components } from '@/api/schema'
+import RoomStatus from '@/components/room/RoomStatus.vue'
+import UserIcon from '@/components/generic/UserIcon.vue'
+import { useUserStore } from '@/store'
+import { computed } from 'vue'
+import { useQueryClient } from '@tanstack/vue-query'
+import { qk } from '@/api/queries/keys'
+
+type Room = components['schemas']['RoomResponse']
+type RoomStatus = components['schemas']['RoomStatus']
```

Biome (2.3.14)

[error] Shouldn't redeclare 'RoomStatus'. Consider to delete it or rename it.

'RoomStatus' is defined here:


(lint/suspicious/noRedeclare)